edxval/migrations/0007_auto_20171005_0154.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we should rename the migration to better name ? just as we did for `006_`